R/facet-.R

```diff
@@ -90,6 +94,7 @@ Facet <- ggproto("Facet", NULL,
   map_data = function(data, layout, params) {
     cli::cli_abort("Not implemented.")
   },
+  setup_panel_params = function(self, panel_params, coord) panel_params,
```

Comment:
Just a little bit of future-proofing
```suggestion
  setup_panel_params = function(self, panel_params, coord, ...) panel_params,
```